### ISSUE TITLE
Request of characters: advanced punctuation

### ISSUE THREAD

**Issue by pereelmagne** [IBM/plex#492]
― U+2015 HORIZONTAL BAR «Punctuation, Dash»
‼︎ U+203C DOUBLE EXCLAMATION MARK
⁇ U+2047 DOUBLE QUESTION MARK
⁈ U+2048 QUESTION EXCLAMATION MARK
⁉ U+2049 EXCLAMATION QUESTION MARK  
⸻ U+2E3B THREE-EM DASH «Punctuation, Dash»
⸺ U+2E3A TWO-EM DASH «Punctuation, Dash»
⁚ U+205A TWO DOT PUNCTUATION «Punctuation, Other»
⁖ U+2056 THREE DOT PUNCTUATION «Punctuation, Other»
⁘  U+2058 FOUR DOT PUNCTUATION «Punctuation, Other»
⁙ U+2059 FIVE DOT PUNCTUATION «Punctuation, Other»
⁛ U+205B FOUR DOT MARK
⁝ U+205D TRICOLON
⁞ U+205E VERTICAL FOUR DOTS
⁓ U+2053 SWUNG DASH «Punctuation, Other»
‽ U+203D INTERROBANG «Punctuation, Other»
‖ U+2016 DOUBLE VERTICAL LINE «Punctuation, Other»

**Comment by fitojb** [comment]
+1 for HORIZONTAL BAR and THREE-EM DASH, which I use constantly.

U+2015 wraps lines correctly in Spanish, unlike U+2014 which behaves in a manner more appropriate for the way it’s used in English.

And U+2E3B is very useful for bibliographies, to indicate that an author is repeated on the list.

**Comment by joergklein** [comment]
Please add

    U+2009 Thin Space
    U+2011 Non-Breaking Hyphen

**Comment by BoldMonday** [comment]
@joergklein 
> ```
> U+2009 Thin Space
> U+2011 Non-Breaking Hyphen
> ```

These glyphs are already included in the latest versions of Plex Sans, Plex Sans Condensed and Plex Serif.
Please see https://github.com/IBM/plex/releases

**Comment by real-or-random** [comment]
> ```
> U+2009 Thin Space
> U+2011 Non-Breaking Hyphen
> ```
> 
> These glyphs are already included in the latest versions of Plex Sans, Plex Sans Condensed and Plex Serif. Please see [IBM/plex/releases](https://github.com/IBM/plex/releases)

That's not true for the Condensed variant. It seems that some (all?) the recently characters weren't added to the Condensed. A workaround is to use the variable one.

**Comment by BoldMonday** [comment]
> That's not true for the Condensed variant. It seems that some (all?) the recently characters weren't added to the Condensed. A workaround is to use the variable one.

Strange. And thanks for spotting. Those updated Plex Sans Condensed fonts have been produced already but somehow never made it to this repositry. Just reported it as an [issue](https://github.com/IBM/plex/issues/557)

**Comment by BoldMonday** [comment]
@real-or-random latest version of Plex Sans Condensed has been released with https://github.com/IBM/plex/releases/tag/v6.4.0
